### PR TITLE
Ble integration

### PR DESCRIPTION
**Pull request by theScrabi**
A try of implementing the bluetooth comunication.
This PR uses the RPIs generated by the [CryptoModule](https://github.com/CoraLibre/CoraLibre-android-sdk/blob/10ee5f147ad899bf61a36f9029c44f27e9b1259e/coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/crypto/ppcp/CryptoModule.java) and sends them. On the other hand it receives RPIs and saves them to the database.

TODO:
- [ ] Write Tests
- [ ] Double check against [the documentation](https://covid19-static.cdn-apple.com/applications/covid19/current/static/contact-tracing/pdf/ExposureNotification-BluetoothSpecificationv1.2.pdf) if parameters are set correctly
- [ ] reviews
- [ ] refactor names
- [ ] refactor ennumber constructor (`new ENNumber -> ENNumber.fromUnixTime()...`)
- [x] implement [ScanInstance](https://developers.google.com/android/exposure-notifications/ble-attenuation-overview)
- [x] scan every five minutes for four seconds

### CODE REVIEW

File: coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 import java.util.List;
 
-public interface IntervalOfCollectedPackages {
+public interface IntervallOfCapturedData {
```
Comment (Stypox) [inline coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java:7]:
```suggestion
public interface IntervalOfCapturedData {
```

File: coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedDataImpl.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.coralibre.android.sdk.internal.database.ppcp.model;
+
+import org.coralibre.android.sdk.internal.crypto.ppcp.ENNumber;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IntervallOfCapturedDataImpl implements IntervallOfCapturedData {
```
Comment (Stypox) [inline coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedDataImpl.java:8]:
```suggestion
public class IntervalOfCapturedDataImpl implements IntervallOfCapturedData {
```

File: coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/CapturedData.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package org.coralibre.android.sdk.internal.database.ppcp.model;
+
+public class CapturedData {
+    private long captureTimestamp; // in milliseconds since Epoch
+
+    /** in db */
```
Comment (Stypox) [inline coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/CapturedData.java:6]:
```suggestion
    /** in dB */
```

File: coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java (hunk 1 above)
Comment (theScrabi) [reply coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java:7]: xD who do you think I am. Of cource Typos :P

Comment (theScrabi) [reply coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java:7]: No for real I'll fix it.

Comment (Stypox) [reply coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/database/ppcp/model/IntervallOfCapturedData.java:7]: I meant, I still don't have the knowledge to really review the code ;-D

**Pull request merged**